pumps/prometheus.go

```diff
@@ -400,6 +408,21 @@ func (pm *PrometheusMetric) GetLabelsValues(decoded analytics.AnalyticsRecord) [
 	return values
 }
 
+func obfuscateAPIKey(apiKey string, obfuscate bool, length int) string {
```

Comment:
also, if we only use this function in prometheus can we just change the signature to something like:

`func(pm *PrometheusMetric) obsfusctaeKey(key string){}` ?